test/python/circuit/test_circuit_properties.py

```diff
@@ -59,6 +60,13 @@ def test_qarg_numpy_int_exception(self):
         """
         self.assertRaises(CircuitError, QuantumCircuit, 'string')
 
+    def test_warning_on_noninteger_float(self):
+        """Test warning when passing non-integer float to QuantumCircuit
+        """
+        self.assertRaises(CircuitError, QuantumCircuit, 2.2)
+        # but an integer float should pass
+        QuantumCircuit(2.0)
```

Comment:
Can you add a simple test here (also to the other one below)? Something like
```
qc = QuantumCircuit(2.0)
self.assertEqual(qc.num_qubits, 2)
```

Reply:
I have now added this to testing both `QuantumCircuit` and `QuantumRegister`.